exercises/J-functions/exercise.js

```diff
@@ -1,5 +1,7 @@
 function halve(number) {
   // complete the function here
+   var result = (number / 2);
+   return result 
```

Comment:
Hi Bisi, great work, well done.
As a small suggestion: Even if the program works we always need to use semicolon at the end of the statement.  (eg:return result;) 
Please look at here https://www.w3schools.com/js/js_conventions.asp

Reply:
Thanks Adem